bigbluebutton-html5/imports/ui/components/chat/message-form/component.jsx

```diff
@@ -153,7 +153,7 @@ class MessageForm extends Component {
             label={intl.formatMessage(messages.submitLabel)}
             hideLabel
             icon="send"
-            onClick={() => null}
+            onClick={() => this.hangleMessageKeyDown}
```

Comment:
This 'this.hangleMessageKeyDown' need to be changed. There is no error on the console because of this, but I can't find 'hangleMessageKeyDown'.